documents/operating.md

```diff
@@ -13,10 +13,44 @@ Contents:
 ---------------------------------------------------------------------
 SECTION 1: 📚📚📚 Official Operating Agreement 📚📚📚
 ---------------------------------------------------------------------
-- [ ] Company term
-- [ ] Continuance of company
-- [ ] Rickroll
-- [ ] Game of Mario Party 3 before legal arbitration between members
+- [x] Company term
+- [x] Continuance of company
+- [x] Rickroll
+- [x] Game of Mario Party 3 before legal arbitration between members
+
```

Comment:
Shelving this for the time being, as it probably doesn't belong in Sections 0 or 1, but maybe it should be its own new Section 2, or 1.π